modules/flowable-cmmn-rest/src/main/java/org/flowable/cmmn/rest/service/api/runtime/task/TaskIdentityLinkResource.java

```diff
@@ -72,9 +79,15 @@ public void deleteIdentityLink(@ApiParam(name = "taskId") @PathVariable("taskId"
             @ApiParam(name = "type") @PathVariable("type") String type,
             HttpServletResponse response) {
 
-        Task task = getTaskFromRequest(taskId);
+        Task task = taskService.createTaskQuery().taskId(taskId).singleResult();
+        if (task == null) {
+            throw new FlowableObjectNotFoundException("Could not find a task with id '" + taskId + "'.", Task.class);
+        }
 
         validateIdentityLinkArguments(family, identityId, type);
+        if (restApiInterceptor != null) {
+            restApiInterceptor.deleteTaskIdentityLink(task, identityId, family, type);
```

Comment:
Same story as for deleting an identity link of a case instance

Reply:
done